COMMERCE-9816 - The value of an option disappears from the UI when a value too high is assigned to the value position

victorcdp opened the pull request:
The priority input number for product options had no max value, causing problems with the table display when the priority was saved with a very high number.
Proposed fix is to add checks at the front and back end, only allowing the input number to be lower than an integer max value

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brianikim commented:
ci:test:relevant

brianikim commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest(master)/3899/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-22.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3899/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-17.liferay.com/job/test-portal-source-format/1510/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-17.liferay.com//userContent/jobs/test-portal-source-format/builds/1510/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes 2 seconds 98 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7d70691b1dd821684bed4945dfdaecad6fa3aada">7d70691b1dd821684bed4945dfdaecad6fa3aada</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/matthewchan123/liferay-portal/tree/LPS-93908">LPS-93908</a><br/>Branch GIT ID: <a href="https://github.com/matthewchan123/liferay-portal/commit/de54a49e75cac1290a07366acb29c217382aa9b9">de54a49e75cac1290a07366acb29c217382aa9b9</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-source-format/1510/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/1510/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 19 out of 21 jobs passed in 1 hour 19 minutes 19 seconds 388 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/340efa0451d2dc117a2b3b6af6490a27b75856d5">340efa0451d2dc117a2b3b6af6490a27b75856d5</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/4ded2b18ef9cb71a46ebdf9afa18b0bfc67d510e">4ded2b18ef9cb71a46ebdf9afa18b0bfc67d510e</a></p>19 out of 21 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest(master)/3899/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308002/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</strong></strike></a></li></ul></div><details><summary><strong>19 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/285129/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201101/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201104/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/250263/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/158183/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201110/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/250264/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/250265/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201258/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201111/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/158184/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308003/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201112/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308004/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/250266/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/294065/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201259/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/158185/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/250267/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-22.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3899/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/308002/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/308002//consoleText">AXIS_VARIABLE=0,label_exp=!master #308002</a><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/308002//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchive621021_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive621021</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-22/1555093721460/test-portal-acceptance-pullrequest(master)/3899/functional-upgrade-tomcat90-mysql57-jdk8/2/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive621021/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-22/1555093721460/test-portal-acceptance-pullrequest(master)/3899/functional-upgrade-tomcat90-mysql57-jdk8/2/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive621021/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-22/1555093721460/test-portal-acceptance-pullrequest(master)/3899/functional-upgrade-tomcat90-mysql57-jdk8/2/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
Reviewing now :)

brianikim commented:
https://github.com/SamZiemer/liferay-portal/pull/580

ChrisKian committed:
COMMERCE-9441 Apply patched library

Commit:
LPS-151786 Creating the file and adding stubs for CP2NewKeyGenerationPortalPart2

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmStructureService,as its usages was removed in 8ba6c233960d8d46dbab63e096937debbd7c3cb5.

JanisZhang committed:
LPS-165338 Remove unused Reference field _jsonFactory, its usages was removed in 48120813fc1944d0fe53e0672ecd909abdf4c240.

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmFormFieldTypeServicesTracker, its usages was removed in 4de6ae1e80dbb17f8571d0a955dd6ee55d7cd9b2.

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmDataProviderInstanceService, its usages was removed in 80c91060737dca8eb6882f39520c38164f5ff934.

JanisZhang committed:
LPS-165338 Make JSONFactory field used; although the direct usages was removed in af07059a739eae579e9e4f2bcb7900101c1c4cb3, the JSONFactoryUtil usage by the class it creates and uses should still be protected by the reference.

JanisZhang committed:
LPS-165338 Remove unused Reference field _jsonDDMFormDeserializer, its usages was removed in 5a93ea812ae6e329d2078d510a80f0aef3ff6de8.

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmFormInstanceLocalService, as it already been referenced in DDMFormInstanceUADDisplay's parent class.

CrescenzoRegaLF committed:
COMMERCE-6413 Add type validation

CrescenzoRegaLF committed:
COMMERCE-6413 Add new exception mappers

Commit:
LPS-163599 - add disableFlag property in InputLocalized

slnn committed:
LPS-166048 Remove reference field _userLocalService from JournalArticleContentDashboardItemFactory, FileEntryContentDashboardItemFactory and BlogsEntryContentDashboardItemFactory

Note:
Field _userLocalService in JournalArticleContentDashboardItemFactory was added to the class and applied usage by LPS-117280:eb7109e8f6ad3c84ddc1e723eed36518685c80cf, its usage was removed by LPS-119190:f5ac553d55984c9e8994837779bb5b29973f9b09.

Field _userLocalService in FileEntryContentDashboardItemFactory and BlogsEntryContentDashboardItemFactory was added to these classes when creating these classes but no one using it

slnn committed:
LPS-166048 Remove reference fields _sharingConfigurationFactory and _sharingPermission from class ContentDashboardAdminPortlet since no one uses them directly in the class

slnn committed:
LPS-166048 Remove reduant reference field _classNameLocalService from class ContentDashboardAdminPortlet since the field is not used directly in the class and the class's another field reference _contentDashboardItemFactoryTracker has declared a field reference for ClassNameLocalService

slnn committed:
LPS-166048 Remove reference field _assetVocabularyLocalService from UpdateContentDashboardConfigurationMVCActionCommand since it is added and applyied usage by LPS-128990:7a5674d57504739eb8c98d5ab52504c602fb7e03 but usage was removed by LPS-128990:24bf462165aca405436a4a604d354cf9865ef45f

CrescenzoRegaLF committed:
COMMERCE-6413 Update usages

DiegoHu97 committed:
LPS-165480 Quit editable config when is not mapped

DiegoHu97 committed:
LPS-165480 Change default value to adapt it to localized date formats

DiegoHu97 committed:
LPS-164806 Adapt to localized date formats

DiegoHu97 committed:
LPS-164806 Add language flag to dropdown menu

victoroscuro committed:
COMMERCE-6413 Create testcase and fix existing macro to create channels

Commit:
LPS-151786 Add generateKeyFlow macro

Commit:
LPS-151786 Add IfGenerate1KeyIsEnable test

Commit:
LPS-151786 Add IfAddActivationIsDisabledWithMaximumNumber test

Commit:
LPS-151786 Add IfUserCanAddMoreIPAddresses test

Commit:
LPS-151786 Add IfUserCanAddMoreMACAddresses test

Commit:
LPS-151786 Add IfRemoveActivationKeyIsWorking test

Commit:
LPS-151786 Add IfGenerateXKeysIsDisabled test

Commit:
LPS-151786 Auto SF

JanisZhang committed:
LPS-166022 Remove unused reference field _journalContent, its usage was removed in 2e51e57644e579e95cf55b7d80f5f89a50125fff

JanisZhang committed:
LPS-166022 Remove unused reference field _infoItemServiceTracker, it was added in 2eaaeb6a0d3ed0010e003de87594d54f2bdd2028 but never used

JanisZhang committed:
LPS-166022 Remove unused reference field _fieldsToDDMFormValuesConverter, its usage was removed in 7a4d5a6cf6472d8eb7e1bb4a772f6db6e822149b

JanisZhang committed:
LPS-166022 Remove unused reference fields, they have been referenced in JournalHelperImpl

JanisZhang committed:
LPS-166022 Remove unused reference field _groupService, it was added in 2700f809d4872e3c446d8580d09e479fe41b38e6 but never used

JanisZhang committed:
LPS-166022 Remove unused reference field _groupLocalService, its usage was removed in bbfeb4e51ce31ac61a8483fd8a7f9f9f041804dc

JanisZhang committed:
LPS-166022 Remove unused reference field _journalFolderLocalService, it was added since class creation but never used

JanisZhang committed:
LPS-166022 Remove unused reference field _portal, it has been referenced in DDMStructureLocalServiceImpl .(through _journalArticleService.deleteArticleDefaultValues() ->  _ddmStructureService.getStructure())

JanisZhang committed:
LPS-166022 Remove unused reference field _jsonFactory, it has been referenced in DDMStructureLocalServiceImpl

JanisZhang committed:
LPS-166022 Remove unused reference field _ddmStructureLocalService, its usage has been removed in 4cec2cd8ad58180705984469cb3b7b0aba6903de

JanisZhang committed:
LPS-166022 Remove unused reference field _portal, it has been referenced in JournalHelperImpl

victorg1991 committed:
LPS-164806 Not needed, since we only have to conditions, we can explicitly set those

victorg1991 committed:
LPS-164806 Simplify

no need to use editableValue.config.dateFormat everytime we can extract that to a variable

Commit:
LPS-166034 Remove unused reference variable _portal(It was added when the class was created, but never used(eca1b86c3c0e453421313e9f2ea449c62a71ad53))

Commit:
LPS-166034 Remove unused reference variable _portal(usge was removed in task LPS-89995(c5b3fe5ddf527dcb5a3e781a437dba9345230c06))

CrescenzoRegaLF committed:
COMMERCE-6413 Align the integration test classes

ces-sontruong committed:
LPS-165991 Change parameter order

Commit:
COMMERCE-8863 Using GetterUtil to account for null values

p2kmgcl committed:
LPS-165954 Remove toolbar custom ordering from document-library-web

Commit:
LPS-165677 HR_74 Add aria-label to clay:dropdown-actions taglib call

Commit:
LPS-165701 HR_139 Add title prop to ActionControls' dropdown trigger

rafaelfs98 committed:
LPS-166079 Fixed creating duplicate projects and Routine

Commit:
LPS-151786 Fix CP2Overview paths

Commit:
LPS-151786 Fix CP2SLACard

Commit:
LPS-151786 SF

joelrocha68 committed:
LPS-166133 Fix tests for Testray 2

joelrocha68 committed:
LPS-166133 SF

SelenaAungst committed:
LPS-165966 Added clay alert to utilize autoClose feature.

rafaelfs98 committed:
LPS-166080 Fixed creating duplicate Suites and Cases

Commit:
LPS-151786 Add assertSLACardUnicContentIsVisible macro

Commit:
LPS-151786 Add SLA_CARD_UNIC_CONTENT path

Commit:
LPS-151786 Fix CardsAtTheSameSupportLevelThePreviousDeadlineIsExtended test

SelenaAungst committed:
LPS-165966 Removed unused import.

slnn committed:
LPS-166158 Remove reference field _companyService from PortalInstancesLocalServiceImpl which added to the class when creating class but no directly usage, no tests failed if remove it

andrea-ale-sbarra committed:
COMMERCE-9441 autogenerated code (portal-impl ant build-lib-versions)

ambrinchaudhary committed:
LPS-166066 Add custom error message

ambrinchaudhary committed:
LPS-166066 Add key

ambrinchaudhary committed:
LPS-166066 buildLang

rafaelfs98 committed:
LPS-166081 Fixed creating duplicate Category,Options and Cases Types

DiegoHu97 committed:
LPS-164806 Adapt default option

rafaelfs98 committed:
LPS-166082 Fixed creating duplicate Team

ealonso committed:
LPS-166142 Adds new taglib that includes the cancel and submit buttons in the correct order and with the correct styles to make those accessible, also avoid developers mistakes

ealonso committed:
LPS-166142 Uses new taglib

kevenleone committed:
LPS-166220 Rename Relationship Label

kevenleone committed:
LPS-166221 Adapt Build to Render data from Aggregation

HenriquePereiraa committed:
LPS-165709 [HR_146] Giving anchor tags a buttone label and appropriate style

rafaelfs98 committed:
LPS-166083 Fixed creating duplicate TestFlow

Commit:
LPS-151786 Fix generateKeyFlow macro

Commit:
LPS-151786 Fix IfAddActivationIsDisabledWithMaximumNumber test

Commit:
LPS-151786 Fix IfGenerate1KeyIsEnable test

Commit:
LPS-151786 Fix IfGenerateXKeysIsDisabled test

Commit:
LPS-151786 Fix IfRemoveActivationKeyIsWorking test

Commit:
LPS-151786 Fix IfUserCanAddMoreIPAddresses test

Commit:
LPS-151786 Fix IfUserCanAddMoreMACAddresses test

ozysouza committed:
LPS-164770 Add test case to Form Components should not be available on display page template

ozysouza committed:
LPS-164770 Add testcase to user can map a Form Container on masters page template

ozysouza committed:
LPS-164770 Add testcase to Form Components should available on Masters Page Template for user with custom permissions

Commit:
LPS-151786 Auto SF

brianchandotcom committed:
LPS-165486 SF

brianchandotcom committed:
artifact:ignore commerce-shipping-web 6.0.24 prep next

brianchandotcom committed:
artifact:ignore commerce-shipping-web 6.0.24 artifact properties

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore commerce-tax-web 4.0.18 change log

brianchandotcom committed:
artifact:ignore commerce-tax-web 4.0.18 prep next

brianchandotcom committed:
artifact:ignore commerce-tax-web 4.0.18 artifact properties

brianchandotcom committed:
artifact:ignore commerce-term-web 1.0.22 change log

brianchandotcom committed:
artifact:ignore commerce-term-web 1.0.22 prep next

brianchandotcom committed:
artifact:ignore commerce-term-web 1.0.22 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell-site-initializer 4.0.20 change log

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell-site-initializer 4.0.20 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell-site-initializer 4.0.20 artifact properties

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.64 change log

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.64 prep next

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.64 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-cart-content-web 4.0.34 change log

brianchandotcom committed:
artifact:ignore commerce-cart-content-web 4.0.34 prep next

brianchandotcom committed:
artifact:ignore commerce-cart-content-web 4.0.34 artifact properties

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.56 change log

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.56 artifact properties

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.56 prep next

brianchandotcom committed:
artifact:ignore commerce-inventory-web 4.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-inventory-web 4.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-inventory-web 4.0.32 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 4.0.37 change log

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 4.0.37 prep next

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 4.0.37 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-content-web 4.0.67 change log

brianchandotcom committed:
artifact:ignore commerce-product-content-web 4.0.67 prep next

brianchandotcom committed:
artifact:ignore commerce-product-content-web 4.0.67 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-ddm-impl 4.0.16 change log

brianchandotcom committed:
artifact:ignore commerce-product-ddm-impl 4.0.16 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-ddm-impl 4.0.16 prep next

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.25 change log

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.25 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.25 prep next

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.36 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.36 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.36 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 4.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 4.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 4.0.32 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fedex 5.0.23 change log

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fedex 5.0.23 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fedex 5.0.23 prep next

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-web 4.0.35 change log

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-web 4.0.35 prep next

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-web 4.0.35 artifact properties

brianchandotcom committed:
artifact:ignore commerce-subscription-web 4.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-subscription-web 4.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-subscription-web 4.0.32 artifact properties

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.34 change log

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.34 prep next

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.34 artifact properties

brianchandotcom committed:
artifact:ignore commerce-wish-list-web 4.0.30 change log

brianchandotcom committed:
artifact:ignore commerce-wish-list-web 4.0.30 prep next

brianchandotcom committed:
artifact:ignore commerce-wish-list-web 4.0.30 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium-impl 5.0.19 change log

brianchandotcom committed:
artifact:ignore commerce-theme-minium-impl 5.0.19 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-minium-impl 5.0.19 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 5.0.27 change log

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 5.0.27 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 5.0.27 artifact properties

brianchandotcom committed:
build.gradle auto SF

Commit:
LPS-166122 ComparisonFailure in ObjectEntryLocalServiceTest.testUpdateObjectEntry

brianchandotcom committed:
artifact:ignore analytics-reports-blogs-impl 1.0.2 change log

brianchandotcom committed:
artifact:ignore analytics-reports-blogs-impl 1.0.2 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-blogs-impl 1.0.2 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-impl 2.0.6 change log

brianchandotcom committed:
artifact:ignore analytics-reports-impl 2.0.6 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-impl 2.0.6 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-layout-display-page-impl 1.0.11 change log

brianchandotcom committed:
artifact:ignore analytics-reports-layout-display-page-impl 1.0.11 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-layout-display-page-impl 1.0.11 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-layout-impl 1.0.10 change log

brianchandotcom committed:
artifact:ignore analytics-reports-layout-impl 1.0.10 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-layout-impl 1.0.10 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.62 change log

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.62 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.62 artifact properties

brianchandotcom committed:
artifact:ignore commerce-machine-learning-impl 5.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-machine-learning-impl 5.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-machine-learning-impl 5.0.32 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-dxp-override 4.0.35 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-dxp-override 4.0.35 prep next

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.28 change log

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.28 prep next

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.28 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.58 change log

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.58 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.58 artifact properties

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.19 change log

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.19 prep next

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.19 artifact properties

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.38 change log

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.38 prep next

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.38 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.20 change log

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.20 prep next

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.20 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.19 change log

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.19 prep next

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.19 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.36 change log

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.36 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.36 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.27 change log

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.27 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.27 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore saml-web 5.0.41 change log

brianchandotcom committed:
artifact:ignore saml-web 5.0.41 prep next

brianchandotcom committed:
artifact:ignore saml-web 5.0.41 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
7.4.3.48 GA48 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bookmarks, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream

brianchandotcom committed:
7.4.13 Update 48 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bookmarks, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream, antivirus, enterprise-product-notification, multi-factor-authentication, portal-search-elasticsearch-cross-cluster-replication, portal-search-elasticsearch-monitoring, portal-search-learning-to-rank, portal-search-similar-results, portal-search-tuning, saml, search-experiences

ling-alan-huang committed:
LPS-163638 Fix missing language key

CrescenzoRegaLF committed:
COMMERCE-6413 Remove unused InvalidCommerceChannelTypeException class

CrescenzoRegaLF committed:
COMMERCE-6413 Rename InvalidCommerceChannelType in CommerceChannelType (services.xml)

CrescenzoRegaLF committed:
COMMERCE-6413 Autogenerated code (blade gw buildService)

CrescenzoRegaLF committed:
COMMERCE-6413 Semantic versioning (gw baseline)

CrescenzoRegaLF committed:
COMMERCE-6413 Update the use of the exception

victorg1991 committed:
LPS-165966 Use openToast for the screen reader to read the alert. Also focus the clay alert to make it easy for the user to use the actions

p2kmgcl committed:
LPS-165954 Move top/height sidebar styles to MultiPanelSidebar

p2kmgcl committed:
LPS-165954 Fix toolbar wrap in desktop size

brianchandotcom committed:
LPS-166066 Wordsmith

brianchandotcom committed:
LPS-166066 Regen

CrescenzoRegaLF committed:
COMMERCE-6413 Autogenerated code (blade gw buildREST)

liferay-translation committed:
LPS-77699 Update Translations

victorg1991 committed:
LPS-165966 Language keys

victorg1991 committed:
LPS-165966 Replace keys

victorg1991 committed:
LPS-165966 buildLang

victorg1991 committed:
LPS-165966 Extract to js file

adolfopa committed:
LPS-166292 Update file store README with preview/thumbnail behavior

carolmariaabb committed:
LPS-165991 Auto SF

brianchandotcom committed:
Revert "LPS-165486 SF"

This reverts commit 6def0503a33e73bc1db099e46032140201da5c5f.

brianchandotcom committed:
COMMERCE-8863 SF

Luizcarlosqueiroz committed:
LRQA-78099 Fix value for assert

Luizcarlosqueiroz committed:
LRQA-78100 Add FF property

kevenleone committed:
LPS-166245 Reset Breadcrumb List when no project is selected

kevenleone committed:
LPS-166245 Add Comma

Luizcarlosqueiroz committed:
LRQA-78102 Change changeExperience macro

brianchandotcom committed:
COMMERCE-6413 Name it after the REST name

joelrocha68 committed:
LPS-166133 test corrections

brianchandotcom committed:
LPS-165709 SF

kevenleone committed:
LPS-166245 Fix CaseResultHeader bug when open page

kevenleone committed:
LPS-166323 Add NoWrap|Wrap Property to Table

kevenleone committed:
LPS-166323 Prepare Testflow to Use Aggregation

kevenleone committed:
LPS-166323 Add text-gray Util Class

brianchandotcom committed:
LPS-165966 #sfme

brianchandotcom committed:
LPS-166221 Fix typo

brianchandotcom committed:
LPS-166083 upload

brianchandotcom committed:
LPS-166292 Wordsmith

ealonso committed:
LPS-166324 We should populate it depending on the type, my mistake

Commit:
COMMERCE-9816 - (commerce-product-service) - add checks to see if priority input number is bigger than integer max value

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
